docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md

```diff
@@ -50,11 +48,19 @@ Only a subset of account types and deployment targets support being created dyna
 Before you can create dynamic targets in an Environment, the environment needs to be configured to allow it. See [Enabling dynamic infrastructure](/docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md#enable-dynamic-infrastructure) for more information.
 :::
 
+:::hint
+**Targets from Step Packages**
+Octopus Deploy has recently developed a new architecture for deployment steps and targets, known as step packages. To learn more about step packages, read the [step package documentation](https://github.com/octopusdeploy/step-api#overview).
```

Comment:
Does this need to include the version these became available (2021.2 / 3?)

Reply:
That is a good idea, I'll add that.